lib/manager.js

```diff
@@ -12,22 +12,18 @@ module.exports = class AssetManager {
 		this.fingerprint = fingerprint;
 		this.exitOnError = exitOnError;
 
-		// bind methods for convenience
-		this.writeFile = this.writeFile.bind(this);
-		this.resolvePath = this.resolvePath.bind(this);
+		this.writeFile = this.writeFile.bind(this); // for convenience
+		this.resolvePath = filepath => resolvePath(filepath, referenceDir);
```

Comment:
I opted to replace the class-level method definition with this dynamic definition because it would have to be bound anyway (i.e. for efficiency) - not sure whether that's sensible though because that method's now a non-obvious part of the interface.